src/kern/exec.c

```diff
@@ -16,44 +16,42 @@
 #include "vmlinux.h"
 #include <bpf/bpf_helpers.h>
 #include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+#include <linux/limits.h>
+#include <process_info.h>
```

Comment:
This is a local file, you should use quotes instead of <> (same comment for user.c)